app/screens/wallet/SendCoins.js

```diff
@@ -82,9 +83,11 @@ class SendCoins extends Component<Props, State> {
       currentAccount: { balance },
       fiatRate,
       contacts,
-      lastUsedAddresses
+      transactions
     } = this.props;
     const { address, defaultAddress, amount, addressErrorMsg, amountErrorMsg, feeIndex, note, shouldShowModal, shouldShowContactsModal } = this.state;
+    const lastUsedAddresses: TxList = uniqBy(transactions, 'address').slice(0, 3);
```

Comment:
same comment as before for getting last used addresses

Reply:
ok